yarn.lock

```diff
@@ -24330,6 +24330,13 @@ react-map-gl@7.0.15:
   dependencies:
     "@types/mapbox-gl" "^2.6.0"
 
+react-map-gl@7.0.23:
+  version "7.0.23"
+  resolved "https://registry.yarnpkg.com/react-map-gl/-/react-map-gl-7.0.23.tgz#f6a2daf34fc3e4fb67d7ab4fa687cf6887c8810c"
+  integrity sha512-874jEtdS/fB2R4jSJKud9va0H0GlxhtiSFuUMATiniQ7A2lQnZLkZIPEWwIPkMmNZDXNlTAkxWEdSHzsqADVAw==
+  dependencies:
+    "@types/mapbox-gl" "^2.6.0"
+
```

Comment:
Please double check `yarn.lock` file as this block of code should be replacing the `react-map-gl@7.0.15` block above.

Reply:
We can't have 2 versions of `react-map-gl`

Reply:
I'm not really sure how I did that.. but I reran `yarn install` and it fixed it :)